flycheck.el

```diff
@@ -2191,7 +2191,10 @@ current buffer as BUFFER.
 
 Return non-nil if the BUFFER is backed by a file, and not
 modified, or nil otherwise."
-  (and (buffer-file-name buffer) (not (buffer-modified-p buffer))))
+  (let ((file-name (buffer-file-name buffer))
+        (not-buffer-modified (not (buffer-modified-p buffer))))
```

Comment:
As far as I can see this variable is just used once, right?  So why not inline it?